packages/react/src/components/Toolbar/ToolbarItem.tsx

```diff
@@ -255,6 +255,7 @@ class ToolbarItem extends UIComponent<WithAsProp<ToolbarItemProps>> {
       return Popup.create(popup, {
         defaultProps: {
           trapFocus: true,
+          inline: true,
```

Comment:
As the best practices say, "Don't use trapFocus for inline popup, as it leads to broken behavior for screen reader users."

Reply:
Fixed, using not inlined Popup, and the behavor is as it was before.